web/src/components/navbar/NavBarDesktop.tsx

```diff
@@ -73,7 +75,16 @@ export const NavBarDesktop = (): ReactElement => {
     <div className="position-sticky top-0 z-500 bg-white">
       <nav aria-label="Primary" className="grid-container-widescreen desktop:padding-x-7">
         <div className="display-flex flex-row flex-justify flex-align-center height-8">
-          <NavigatorLogo />
+          <div className="display-flex flex-row flex-align-center">
+            <NavigatorLogo />
```

Comment:
I agree with JP's comment. Since we're rebranding, let's just do away with the `Navigator` prefix. I think `MyAccountLogo` is suitable.